[2024-03-12T00:40:10.661-0400] {processor.py:161} INFO - Started process (PID=299682) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:40:10.663-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T00:40:10.665-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:40:10.664-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:40:10.671-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:40:10.670-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T00:40:10.671-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:40:10.671-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T00:40:10.672-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:40:10.690-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2024-03-12T00:51:15.933-0400] {processor.py:161} INFO - Started process (PID=302379) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:51:15.983-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T00:51:15.986-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:51:15.985-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:51:15.993-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:51:15.991-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T00:51:15.995-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:51:15.994-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T00:51:15.995-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:51:16.014-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2024-03-12T00:52:32.720-0400] {processor.py:161} INFO - Started process (PID=302743) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:52:32.770-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T00:52:32.773-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:52:32.772-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:52:32.781-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:52:32.780-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T00:52:32.782-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:52:32.782-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T00:52:32.782-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:52:32.804-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.090 seconds
[2024-03-12T00:53:39.878-0400] {processor.py:161} INFO - Started process (PID=303099) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:53:39.881-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T00:53:39.882-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:53:39.881-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:53:39.888-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:53:39.887-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T00:53:39.889-0400] {logging_mixin.py:188} INFO - [2024-03-12T00:53:39.888-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T00:53:39.889-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T00:53:39.912-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2024-03-12T01:00:50.537-0400] {processor.py:161} INFO - Started process (PID=304893) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:00:50.588-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:00:50.591-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:00:50.590-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:00:50.597-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:00:50.596-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:00:50.598-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:00:50.598-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:00:50.599-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:00:50.620-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2024-03-12T01:19:55.196-0400] {processor.py:161} INFO - Started process (PID=312053) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:19:55.247-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:19:55.250-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:19:55.249-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:19:55.257-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:19:55.256-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:19:55.258-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:19:55.258-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:19:55.260-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:19:55.280-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.090 seconds
[2024-03-12T01:21:41.760-0400] {processor.py:161} INFO - Started process (PID=312857) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:21:41.764-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:21:41.766-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:21:41.765-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:21:41.773-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:21:41.772-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:21:41.773-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:21:41.773-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:21:41.774-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:21:41.793-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-03-12T01:23:10.853-0400] {processor.py:161} INFO - Started process (PID=313521) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:23:10.905-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:23:10.907-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:23:10.907-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:23:10.913-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:23:10.913-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:23:10.914-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:23:10.914-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:23:10.914-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:23:11.110-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.264 seconds
[2024-03-12T01:24:39.270-0400] {processor.py:161} INFO - Started process (PID=314180) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:24:39.273-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:24:39.275-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:24:39.274-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:24:39.283-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:24:39.282-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:24:39.284-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:24:39.284-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:24:39.284-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:24:39.308-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2024-03-12T01:26:46.115-0400] {processor.py:161} INFO - Started process (PID=315135) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:26:46.124-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:26:46.127-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:26:46.126-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:26:46.133-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:26:46.132-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:26:46.134-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:26:46.133-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:26:46.134-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:26:46.180-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-03-12T01:28:06.712-0400] {processor.py:161} INFO - Started process (PID=315735) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:28:06.746-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:28:06.750-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:28:06.749-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:28:06.763-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:28:06.761-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:28:06.764-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:28:06.763-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:28:06.765-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:28:06.788-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.086 seconds
[2024-03-12T01:29:24.189-0400] {processor.py:161} INFO - Started process (PID=316312) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:29:24.223-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:29:24.228-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:29:24.226-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:29:24.241-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:29:24.236-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:29:24.242-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:29:24.242-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:29:24.243-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:29:24.262-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.080 seconds
[2024-03-12T01:30:34.199-0400] {processor.py:161} INFO - Started process (PID=316815) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:30:34.233-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:30:34.235-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:30:34.234-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:30:34.242-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:30:34.241-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:30:34.243-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:30:34.243-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:30:34.244-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:30:34.261-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-03-12T01:39:05.855-0400] {processor.py:161} INFO - Started process (PID=320195) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:39:05.858-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:39:05.860-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:39:05.859-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:39:05.866-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:39:05.865-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:39:05.866-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:39:05.866-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:39:05.867-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:39:05.890-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2024-03-12T01:40:27.252-0400] {processor.py:161} INFO - Started process (PID=320792) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:40:27.256-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:40:27.259-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:40:27.258-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:40:27.267-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:40:27.266-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:40:27.268-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:40:27.267-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:40:27.268-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:40:27.304-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2024-03-12T01:41:50.323-0400] {processor.py:161} INFO - Started process (PID=321402) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:41:50.358-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:41:50.361-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:41:50.360-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:41:50.368-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:41:50.367-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:41:50.368-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:41:50.368-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:41:50.369-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:41:50.389-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2024-03-12T01:43:22.022-0400] {processor.py:161} INFO - Started process (PID=322089) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:43:22.054-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:43:22.056-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:43:22.056-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:43:22.062-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:43:22.061-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:43:22.063-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:43:22.063-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:43:22.063-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:43:22.087-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2024-03-12T01:47:49.191-0400] {processor.py:161} INFO - Started process (PID=323806) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:47:49.225-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:47:49.227-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:47:49.227-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:47:49.239-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:47:49.237-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:47:49.241-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:47:49.239-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:47:49.241-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:47:49.258-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2024-03-12T01:49:24.189-0400] {processor.py:161} INFO - Started process (PID=324488) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:49:24.224-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:49:24.226-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:49:24.225-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:49:24.236-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:49:24.234-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:49:24.238-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:49:24.238-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:49:24.238-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:49:24.258-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2024-03-12T01:50:55.784-0400] {processor.py:161} INFO - Started process (PID=324807) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:50:55.821-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:50:55.824-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:50:55.823-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:50:55.830-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:50:55.829-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:50:55.831-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:50:55.831-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:50:55.832-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:50:55.877-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2024-03-12T01:52:12.359-0400] {processor.py:161} INFO - Started process (PID=325080) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:52:12.394-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:52:12.397-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:52:12.396-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:52:12.403-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:52:12.402-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:52:12.404-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:52:12.403-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:52:12.404-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:52:12.421-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-03-12T01:53:30.341-0400] {processor.py:161} INFO - Started process (PID=325352) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:53:30.375-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:53:30.378-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:53:30.377-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:53:30.386-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:53:30.385-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:53:30.387-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:53:30.386-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:53:30.387-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:53:30.445-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.111 seconds
[2024-03-12T01:58:16.229-0400] {processor.py:161} INFO - Started process (PID=326137) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:58:16.264-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:58:16.266-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:58:16.265-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:58:16.272-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:58:16.271-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:58:16.273-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:58:16.273-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:58:16.274-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:58:16.298-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-03-12T01:59:49.821-0400] {processor.py:161} INFO - Started process (PID=326445) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:59:49.824-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T01:59:49.826-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:59:49.825-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:59:49.832-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:59:49.831-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T01:59:49.833-0400] {logging_mixin.py:188} INFO - [2024-03-12T01:59:49.832-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T01:59:49.833-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T01:59:49.850-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-03-12T02:02:55.351-0400] {processor.py:161} INFO - Started process (PID=327054) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:02:55.386-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:02:55.389-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:02:55.388-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:02:55.396-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:02:55.395-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:02:55.397-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:02:55.396-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:02:55.397-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:02:55.422-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-03-12T02:05:17.381-0400] {processor.py:161} INFO - Started process (PID=327518) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:05:17.411-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:05:17.414-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:05:17.413-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:05:17.422-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:05:17.420-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:05:17.423-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:05:17.423-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:05:17.424-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:05:17.446-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-03-12T02:06:47.093-0400] {processor.py:161} INFO - Started process (PID=327833) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:06:47.128-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:06:47.132-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:06:47.130-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:06:47.139-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:06:47.138-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:06:47.140-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:06:47.140-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:06:47.141-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:06:47.165-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-03-12T02:08:04.222-0400] {processor.py:161} INFO - Started process (PID=328102) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:08:04.255-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:08:04.258-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:08:04.257-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:08:04.268-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:08:04.266-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:08:04.269-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:08:04.269-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:08:04.270-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:08:04.293-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-03-12T02:09:29.545-0400] {processor.py:161} INFO - Started process (PID=328393) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:09:29.578-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:09:29.580-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:09:29.579-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:09:29.591-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:09:29.587-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:09:29.593-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:09:29.593-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:09:29.594-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:09:29.614-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2024-03-12T02:10:57.013-0400] {processor.py:161} INFO - Started process (PID=328692) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:10:57.020-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:10:57.022-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:10:57.021-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:10:57.042-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:10:57.040-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:10:57.042-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:10:57.042-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:10:57.043-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:10:57.068-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-03-12T02:12:05.086-0400] {processor.py:161} INFO - Started process (PID=328921) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:12:05.120-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:12:05.122-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:12:05.121-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:12:05.128-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:12:05.127-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:12:05.128-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:12:05.128-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:12:05.130-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:12:05.147-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-03-12T02:13:17.032-0400] {processor.py:161} INFO - Started process (PID=329153) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:13:17.065-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:13:17.068-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:13:17.067-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:13:17.074-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:13:17.073-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:13:17.075-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:13:17.074-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:13:17.075-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:13:17.094-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-03-12T02:14:51.825-0400] {processor.py:161} INFO - Started process (PID=329480) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:14:51.860-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:14:51.862-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:14:51.861-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:14:51.876-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:14:51.874-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:14:51.877-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:14:51.877-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:14:51.879-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:14:51.918-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.098 seconds
[2024-03-12T02:16:09.148-0400] {processor.py:161} INFO - Started process (PID=329745) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:16:09.158-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:16:09.160-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:16:09.159-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:16:09.165-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:16:09.165-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:16:09.166-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:16:09.166-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:16:09.166-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:16:09.184-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-03-12T02:17:16.928-0400] {processor.py:161} INFO - Started process (PID=329977) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:17:16.963-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:17:16.967-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:17:16.965-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:17:16.982-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:17:16.980-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:17:16.983-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:17:16.982-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:17:16.983-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:17:17.026-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.107 seconds
[2024-03-12T02:18:25.672-0400] {processor.py:161} INFO - Started process (PID=330196) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:18:25.677-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:18:25.680-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:18:25.679-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:18:25.686-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:18:25.685-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:18:25.687-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:18:25.686-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:18:25.687-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:18:25.706-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2024-03-12T02:19:42.592-0400] {processor.py:161} INFO - Started process (PID=330460) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:19:42.626-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:19:42.629-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:19:42.628-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:19:42.635-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:19:42.635-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:19:42.636-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:19:42.636-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:19:42.637-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:19:42.658-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2024-03-12T02:20:54.554-0400] {processor.py:161} INFO - Started process (PID=330693) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:20:54.588-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:20:54.590-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:20:54.590-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:20:54.596-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:20:54.596-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:20:54.597-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:20:54.597-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:20:54.598-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:20:54.614-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-03-12T02:22:11.738-0400] {processor.py:161} INFO - Started process (PID=330966) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:22:11.772-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:22:11.774-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:22:11.773-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:22:11.780-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:22:11.779-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:22:11.781-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:22:11.781-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:22:11.783-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:22:11.799-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-03-12T02:33:01.854-0400] {processor.py:161} INFO - Started process (PID=332568) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:33:01.865-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:33:01.867-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:33:01.866-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:33:01.873-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:33:01.872-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:33:01.873-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:33:01.873-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:33:01.874-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:33:01.892-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2024-03-12T02:35:59.618-0400] {processor.py:161} INFO - Started process (PID=333139) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:35:59.652-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:35:59.654-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:35:59.653-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:35:59.661-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:35:59.661-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:35:59.662-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:35:59.662-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:35:59.662-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:35:59.679-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-03-12T02:37:18.278-0400] {processor.py:161} INFO - Started process (PID=333411) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:37:18.314-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T02:37:18.317-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:37:18.316-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:37:18.328-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:37:18.326-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T02:37:18.330-0400] {logging_mixin.py:188} INFO - [2024-03-12T02:37:18.329-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T02:37:18.331-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T02:37:18.362-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2024-03-12T14:11:40.742-0400] {processor.py:161} INFO - Started process (PID=336492) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:11:40.745-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:11:40.746-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:11:40.746-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:11:40.748-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:11:40.748-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:11:40.748-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:11:40.748-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:11:40.748-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:11:40.756-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-03-12T14:13:03.924-0400] {processor.py:161} INFO - Started process (PID=336822) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:13:03.928-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:13:03.929-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:13:03.929-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:13:03.932-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:13:03.932-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:13:03.932-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:13:03.932-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:13:03.932-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:13:03.940-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T14:14:14.931-0400] {processor.py:161} INFO - Started process (PID=337142) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:14:14.933-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:14:14.933-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:14:14.933-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:14:14.936-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:14:14.935-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:14:14.936-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:14:14.936-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:14:14.936-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:14:14.944-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:15:04.417-0400] {processor.py:161} INFO - Started process (PID=337339) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:15:04.420-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:15:04.421-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:15:04.420-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:15:04.423-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:15:04.423-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:15:04.424-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:15:04.423-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:15:04.424-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:15:04.431-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T14:16:06.371-0400] {processor.py:161} INFO - Started process (PID=337576) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:16:06.374-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:16:06.375-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:16:06.374-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:16:06.377-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:16:06.376-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:16:06.377-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:16:06.377-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:16:06.377-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:16:06.384-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:17:12.862-0400] {processor.py:161} INFO - Started process (PID=337855) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:17:12.864-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:17:12.865-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:17:12.864-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:17:12.868-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:17:12.868-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:17:12.868-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:17:12.868-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:17:12.869-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:17:12.876-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T14:18:21.716-0400] {processor.py:161} INFO - Started process (PID=338116) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:18:21.719-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:18:21.720-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:18:21.720-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:18:21.722-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:18:21.722-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:18:21.722-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:18:21.722-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:18:21.723-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:18:21.729-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:19:28.923-0400] {processor.py:161} INFO - Started process (PID=338393) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:19:28.924-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:19:28.925-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:19:28.925-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:19:28.927-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:19:28.927-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:19:28.928-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:19:28.928-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:19:28.928-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:19:28.937-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T14:20:31.182-0400] {processor.py:161} INFO - Started process (PID=338640) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:20:31.186-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:20:31.187-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:20:31.187-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:20:31.190-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:20:31.190-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:20:31.190-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:20:31.190-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:20:31.191-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:20:31.214-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-03-12T14:21:37.420-0400] {processor.py:161} INFO - Started process (PID=338914) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:21:37.425-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:21:37.427-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:21:37.426-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:21:37.433-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:21:37.432-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:21:37.433-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:21:37.433-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:21:37.434-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:21:37.453-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2024-03-12T14:22:48.780-0400] {processor.py:161} INFO - Started process (PID=339183) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:22:48.783-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:22:48.784-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:22:48.784-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:22:48.786-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:22:48.786-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:22:48.786-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:22:48.786-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:22:48.787-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:22:48.795-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-03-12T14:24:37.997-0400] {processor.py:161} INFO - Started process (PID=339632) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:24:38.001-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:24:38.001-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:24:38.001-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:24:38.003-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:24:38.003-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:24:38.004-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:24:38.003-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:24:38.004-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:24:38.010-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:25:33.652-0400] {processor.py:161} INFO - Started process (PID=339856) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:25:33.654-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:25:33.654-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:25:33.654-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:25:33.657-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:25:33.657-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:25:33.657-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:25:33.657-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:25:33.657-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:25:33.664-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:26:46.643-0400] {processor.py:161} INFO - Started process (PID=340155) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:26:46.646-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:26:46.647-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:26:46.646-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:26:46.649-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:26:46.649-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:26:46.649-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:26:46.649-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:26:46.649-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:26:46.655-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:27:55.444-0400] {processor.py:161} INFO - Started process (PID=340425) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:27:55.448-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:27:55.448-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:27:55.448-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:27:55.451-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:27:55.450-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:27:55.451-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:27:55.451-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:27:55.451-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:27:55.458-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T14:29:13.774-0400] {processor.py:161} INFO - Started process (PID=340764) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:29:13.776-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:29:13.777-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:29:13.776-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:29:13.779-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:29:13.779-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:29:13.779-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:29:13.779-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:29:13.779-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:29:13.786-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-12T14:30:07.727-0400] {processor.py:161} INFO - Started process (PID=340983) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:30:07.729-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:30:07.730-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:30:07.729-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:30:07.732-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:30:07.731-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:30:07.732-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:30:07.732-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:30:07.732-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:30:07.739-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-12T14:31:18.775-0400] {processor.py:161} INFO - Started process (PID=341260) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:31:18.779-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:31:18.780-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:31:18.779-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:31:18.783-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:31:18.782-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:31:18.783-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:31:18.783-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:31:18.783-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:31:18.793-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-03-12T14:32:32.728-0400] {processor.py:161} INFO - Started process (PID=341567) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:32:32.730-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:32:32.731-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:32:32.731-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:32:32.735-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:32:32.734-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:32:32.735-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:32:32.735-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:32:32.735-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:32:32.744-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T14:34:19.617-0400] {processor.py:161} INFO - Started process (PID=342026) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:34:19.620-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:34:19.621-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:34:19.621-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:34:19.624-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:34:19.624-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:34:19.624-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:34:19.624-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:34:19.624-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:34:19.632-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T14:35:09.295-0400] {processor.py:161} INFO - Started process (PID=342239) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:35:09.299-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:35:09.300-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:35:09.299-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:35:09.302-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:35:09.301-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:35:09.302-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:35:09.302-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:35:09.302-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:35:09.309-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T14:36:10.502-0400] {processor.py:161} INFO - Started process (PID=342509) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:36:10.505-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:36:10.506-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:36:10.505-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:36:10.508-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:36:10.507-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:36:10.508-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:36:10.508-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:36:10.508-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:36:10.515-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T14:37:13.979-0400] {processor.py:161} INFO - Started process (PID=342769) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:37:13.982-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:37:13.985-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:37:13.984-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:37:13.992-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:37:13.991-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:37:13.993-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:37:13.993-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:37:13.994-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:37:14.019-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2024-03-12T14:38:27.843-0400] {processor.py:161} INFO - Started process (PID=343084) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:38:27.848-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T14:38:27.849-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:38:27.848-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:38:27.852-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:38:27.852-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T14:38:27.852-0400] {logging_mixin.py:188} INFO - [2024-03-12T14:38:27.852-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T14:38:27.852-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T14:38:27.861-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-03-12T17:43:49.493-0400] {processor.py:161} INFO - Started process (PID=369633) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:43:49.496-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T17:43:49.496-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:43:49.496-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:43:49.499-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:43:49.498-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T17:43:49.499-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:43:49.499-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T17:43:49.499-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:43:49.506-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-12T17:44:46.937-0400] {processor.py:161} INFO - Started process (PID=369910) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:44:46.941-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T17:44:46.942-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:44:46.942-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:44:46.946-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:44:46.946-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T17:44:46.947-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:44:46.947-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T17:44:46.947-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:44:46.957-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-03-12T17:45:38.383-0400] {processor.py:161} INFO - Started process (PID=370160) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:45:38.386-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T17:45:38.387-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:45:38.387-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:45:38.390-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:45:38.389-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T17:45:38.390-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:45:38.390-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T17:45:38.390-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:45:38.398-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T17:47:15.931-0400] {processor.py:161} INFO - Started process (PID=370666) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:47:15.936-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T17:47:15.937-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:47:15.936-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:47:15.939-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:47:15.939-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T17:47:15.940-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:47:15.940-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T17:47:15.940-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:47:15.947-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T17:48:08.095-0400] {processor.py:161} INFO - Started process (PID=370991) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:48:08.099-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T17:48:08.100-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:48:08.099-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:48:08.103-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:48:08.102-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T17:48:08.103-0400] {logging_mixin.py:188} INFO - [2024-03-12T17:48:08.103-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T17:48:08.103-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T17:48:08.111-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T18:07:04.771-0400] {processor.py:161} INFO - Started process (PID=375525) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:07:04.772-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:07:04.773-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:07:04.773-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:07:04.775-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:07:04.775-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:07:04.776-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:07:04.776-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:07:04.776-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:07:04.782-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-03-12T18:08:00.399-0400] {processor.py:161} INFO - Started process (PID=375789) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:08:00.403-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:08:00.403-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:08:00.403-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:08:00.406-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:08:00.406-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:08:00.406-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:08:00.406-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:08:00.407-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:08:00.414-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T18:08:57.335-0400] {processor.py:161} INFO - Started process (PID=376091) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:08:57.339-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:08:57.339-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:08:57.339-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:08:57.341-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:08:57.341-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:08:57.342-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:08:57.341-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:08:57.342-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:08:57.348-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-12T18:10:03.355-0400] {processor.py:161} INFO - Started process (PID=376393) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:10:03.357-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:10:03.359-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:10:03.358-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:10:03.362-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:10:03.362-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:10:03.362-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:10:03.362-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:10:03.363-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:10:03.371-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T18:11:45.434-0400] {processor.py:161} INFO - Started process (PID=376940) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:11:45.438-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:11:45.439-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:11:45.439-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:11:45.441-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:11:45.441-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:11:45.442-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:11:45.442-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:11:45.442-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:11:45.449-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-03-12T18:12:40.450-0400] {processor.py:161} INFO - Started process (PID=377188) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:12:40.455-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:12:40.456-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:12:40.456-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:12:40.459-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:12:40.459-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:12:40.460-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:12:40.459-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:12:40.460-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:12:40.468-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-03-12T18:14:41.765-0400] {processor.py:161} INFO - Started process (PID=377929) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:14:41.767-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:14:41.768-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:14:41.768-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:14:41.771-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:14:41.771-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:14:41.772-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:14:41.772-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:14:41.772-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:14:41.790-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-03-12T18:15:45.500-0400] {processor.py:161} INFO - Started process (PID=378232) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:15:45.505-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:15:45.505-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:15:45.505-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:15:45.507-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:15:45.507-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:15:45.508-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:15:45.508-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:15:45.508-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:15:45.515-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-03-12T18:16:54.632-0400] {processor.py:161} INFO - Started process (PID=378548) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:16:54.636-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:16:54.638-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:16:54.637-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:16:54.640-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:16:54.640-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:16:54.640-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:16:54.640-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:16:54.640-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:16:54.648-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T18:20:44.095-0400] {processor.py:161} INFO - Started process (PID=379562) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:20:44.099-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:20:44.100-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:20:44.099-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:20:44.104-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:20:44.103-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:20:44.104-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:20:44.104-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:20:44.104-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:20:44.114-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-03-12T18:21:39.050-0400] {processor.py:161} INFO - Started process (PID=379808) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:21:39.053-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:21:39.054-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:21:39.054-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:21:39.057-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:21:39.056-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:21:39.057-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:21:39.057-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:21:39.057-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:21:39.066-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-12T18:22:32.385-0400] {processor.py:161} INFO - Started process (PID=380053) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:22:32.387-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:22:32.388-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:22:32.388-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:22:32.390-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:22:32.390-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:22:32.390-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:22:32.390-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:22:32.391-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:22:32.397-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-12T18:23:43.034-0400] {processor.py:161} INFO - Started process (PID=380388) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:23:43.036-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:23:43.037-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:23:43.037-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:23:43.039-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:23:43.039-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:23:43.040-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:23:43.040-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:23:43.040-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:23:43.051-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-03-12T18:24:49.808-0400] {processor.py:161} INFO - Started process (PID=380681) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:24:49.812-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:24:49.813-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:24:49.812-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:24:49.815-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:24:49.814-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:24:49.815-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:24:49.815-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:24:49.815-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:24:49.822-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T18:26:49.358-0400] {processor.py:161} INFO - Started process (PID=381158) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:26:49.359-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T18:26:49.360-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:26:49.360-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:26:49.363-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:26:49.363-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T18:26:49.364-0400] {logging_mixin.py:188} INFO - [2024-03-12T18:26:49.363-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T18:26:49.364-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T18:26:49.372-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-12T19:40:47.863-0400] {processor.py:161} INFO - Started process (PID=392791) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:40:47.910-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:40:47.912-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:40:47.911-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:40:47.918-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:40:47.917-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:40:47.918-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:40:47.918-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:40:47.919-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:40:47.954-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2024-03-12T19:42:19.843-0400] {processor.py:161} INFO - Started process (PID=393196) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:42:19.894-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:42:19.898-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:42:19.896-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:42:19.908-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:42:19.906-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:42:19.909-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:42:19.908-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:42:19.909-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:42:19.930-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2024-03-12T19:43:42.769-0400] {processor.py:161} INFO - Started process (PID=393546) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:43:42.794-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:43:42.796-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:43:42.796-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:43:42.805-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:43:42.803-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:43:42.806-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:43:42.805-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:43:42.806-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:43:42.869-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.108 seconds
[2024-03-12T19:45:00.961-0400] {processor.py:161} INFO - Started process (PID=393883) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:45:00.964-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:45:00.966-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:45:00.965-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:45:00.973-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:45:00.972-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:45:00.973-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:45:00.973-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:45:00.974-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:45:00.991-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-03-12T19:48:36.239-0400] {processor.py:161} INFO - Started process (PID=394652) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:48:36.291-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:48:36.293-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:48:36.293-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:48:36.300-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:48:36.299-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:48:36.301-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:48:36.301-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:48:36.301-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:48:36.321-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2024-03-12T19:51:57.096-0400] {processor.py:161} INFO - Started process (PID=395450) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:51:57.147-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:51:57.150-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:51:57.149-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:51:57.157-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:51:57.156-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:51:57.158-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:51:57.158-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:51:57.158-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:51:57.440-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.350 seconds
[2024-03-12T19:53:23.566-0400] {processor.py:161} INFO - Started process (PID=395828) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:53:23.587-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:53:23.589-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:53:23.588-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:53:23.597-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:53:23.596-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:53:23.599-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:53:23.598-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:53:23.601-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:53:23.625-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-03-12T19:54:45.335-0400] {processor.py:161} INFO - Started process (PID=396168) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:54:45.387-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:54:45.390-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:54:45.389-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:54:45.407-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:54:45.406-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:54:45.408-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:54:45.408-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:54:45.409-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:54:45.439-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.111 seconds
[2024-03-12T19:55:56.357-0400] {processor.py:161} INFO - Started process (PID=396453) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:55:56.391-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:55:56.395-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:55:56.394-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:55:56.409-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:55:56.407-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:55:56.410-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:55:56.410-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:55:56.411-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:55:56.437-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.086 seconds
[2024-03-12T19:57:19.489-0400] {processor.py:161} INFO - Started process (PID=396792) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:57:19.537-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:57:19.540-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:57:19.539-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:57:19.547-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:57:19.546-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:57:19.548-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:57:19.548-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:57:19.556-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:57:19.587-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.103 seconds
[2024-03-12T19:58:32.006-0400] {processor.py:161} INFO - Started process (PID=397086) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:58:32.046-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:58:32.049-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:58:32.047-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:58:32.064-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:58:32.056-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:58:32.067-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:58:32.067-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:58:32.068-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:58:32.092-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2024-03-12T19:59:47.423-0400] {processor.py:161} INFO - Started process (PID=397402) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:59:47.427-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T19:59:47.434-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:59:47.428-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:59:47.442-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:59:47.440-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T19:59:47.442-0400] {logging_mixin.py:188} INFO - [2024-03-12T19:59:47.442-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T19:59:47.443-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T19:59:47.462-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
